src/js/main.js

```diff
@@ -0,0 +1,8 @@
+import spreadsheet from './spreadsheet';
+import userColsAndRows from './helpers/userColsAndRows';
+
+const spreadsheetContainer = document.querySelector('#spreadsheetContainer');
+
+const [cols, rows] = userColsAndRows();
+
+spreadsheetContainer.append(spreadsheet(cols, rows));
```

Comment:
main.js should be only used for importing files, all the other function and variables should be in its own file.